ecosystem/typescript/sdk/src/utils/api-endpoints.ts

```diff
@@ -0,0 +1,22 @@
+export const NetworkToIndexerAPI: Record<string, string> = {
+  mainnet: "https://indexer.mainnet.aptoslabs.com/v1/graphql",
+  testnet: "https://indexer-testnet.staging.gcp.aptosdev.com/v1/graphql",
+  devnet: "https://indexer-devnet.staging.gcp.aptosdev.com/v1/graphql",
```

Comment:
I didn't realize they weren't consistent @bowenyang007.  Any reason we use non aptoslabs dns for indexers for devnet testnet?

Reply:
This was a @geekflyer decision.